### PR TITLE
uploading rag demo

### PR DESCRIPTION
**Pull request by franciscojavierarceo**

### CODE REVIEW

**Review by HaoXuAI — APPROVED** [review]

File: module_4_rag/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+name = "feast-rag"
+version = "0.1.0"
+description = "a demo using feast to do Retrieval Augmented Generation (RAG)"
+authors = ["franciscojavierarceo"]
+readme = "README.md"
+packages = [{include = "feast_rag"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+feast = "^0.37.0"
```
Comment (HaoXuAI) [inline module_4_rag/pyproject.toml:11]: I bumped the version here

**Comment by HaoXuAI** [comment]
`feast.errors.S3RegistryBucketNotExist: S3 bucket feast-workshop-danny for the Feast registry does not exist`
This seems to be a irrelevant issue. Can we force the merge?

File: module_4_rag/pyproject.toml (hunk 1 above)
Comment (franciscojavierarceo) [reply module_4_rag/pyproject.toml:11]: nice.

**Comment by franciscojavierarceo** [comment]
> `feast.errors.S3RegistryBucketNotExist: S3 bucket feast-workshop-danny for the Feast registry does not exist` This seems to be a irrelevant issue. Can we force the merge?

yeah we can but we don't need to merge until we're ready with the demo

**Pull request merged**